pkg/daemon/update.go

```diff
@@ -125,6 +125,32 @@ func (dn *Daemon) deleteStaleData(oldConfig, newConfig *mcfgv1.MachineConfig) {
 	}
 }
 
+// enableUnit enables a systemd unit via symlink
+func (dn *Daemon) enableUnit(unit ignv2_2types.Unit) error {
+	// The link location
+	wantsPath := filepath.Join(dn.prefix, wantsPathSystemd, unit.Name)
+	// sanity check that we don't return an error when the link already exists
+	if _, err := os.Stat(wantsPath); err != nil {
+		glog.Warningf("%s already exists. Not making a new symlink", wantsPath)
```

Comment:
I'm not sure if these should be warnings. Once the MCD decides it wants to apply the config, it just lays down all the files, systemd units, etc, exactly how they are defined in the new config without really checking the old one, which means that this warning will get spit out for every enabled unit every time a new config is applied, if that makes sense. same with the other function below.

maybe just do `Info`?

Reply:
I'm cool with it being an `Info` instead. I'll amend :-)

Reply:
Updated.